### ISSUE TITLE
Update to 0.16.3 results in my profile pic showing on most chats instead of the other person's

### ISSUE THREAD

**Issue by kethinov** [vector-im/riot-web#7269]
![screen shot 2018-09-03 at 1 45 02 pm](https://user-images.githubusercontent.com/419567/44997951-e6a3c500-af7f-11e8-9842-19777eda2a4a.png)

In the above screenshot, my profile pic is repeated several times in chats with others where it should show their profile pics instead. This doesn't happen in all chats. A handful of chats show the correct profile pic. And chats with users who didn't set profile pics render correctly. This started happening when I updated to 0.16.3 on the macOS Electron version.

**Comment by turt2live** [comment]
This was supposed to be fixed in 0.16.3 :(

Regression of the regression https://github.com/vector-im/riot-web/issues/7235

**Comment by lampholder** [comment]
Yep, sorry about this. We've got a (real) fix inbound - it'll likely be merged and released tomorrow.
